internal/cluster/distribution/eks/eksadapter/node_pool_manager.go

```diff
@@ -16,27 +16,48 @@ package eksadapter
 
 import (
 	"context"
+	"strconv"
+	"strings"
 	"time"
 
 	"emperror.dev/errors"
 	"go.uber.org/cadence/client"
 
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+
 	"github.com/banzaicloud/pipeline/internal/cluster"
 	"github.com/banzaicloud/pipeline/internal/cluster/distribution/eks"
+	"github.com/banzaicloud/pipeline/internal/cluster/distribution/eks/eksprovider/workflow"
 	"github.com/banzaicloud/pipeline/internal/cluster/distribution/eks/eksworkflow"
+	"github.com/banzaicloud/pipeline/internal/global"
+	"github.com/banzaicloud/pipeline/internal/kubernetes"
+	"github.com/banzaicloud/pipeline/pkg/kubernetes/custom/npls"
 )
 
+// CF stack name prefix
+const StackNamePrefix = "pipeline-eks-nodepool-"
+
 type nodePoolManager struct {
-	workflowClient client.Client
-	enterprise     bool
+	workflowClient       client.Client
+	enterprise           bool
+	dynamicClientFactory kubernetes.DynamicClientFactory
+	aWSSessionFactory    workflow.AWSSessionFactory
```

Comment:
```suggestion
	awsSessionFactory    workflow.AWSSessionFactory
```

Reply:
I'd copy the interface instead of importing it.

Reply:
Fixed in 0805a891374b5af9ef3e2af8b4628bccb4284d65 and b7680606f1d23f944d37cfb70a9140b2845814a1 respectively.